### PR TITLE
Getting started tutorial update mrtk 2.7.0

### PR DESCRIPTION
**Pull request by veeruby-developer**
- Updated MRTK Version to 2.7.0
- Updated the tutorials to support Unity 2019 Legacy WSA, XRSDK and Unity 2020 OpenXR and XRSDK.
- Tested Getting Started tutorial series for MRTK 2.7.0 with Unity 2019 Legacy WSA, XRSDK and Unity 2020 OpenXR and XRSDK.
- Screenshots updated.

### CODE REVIEW

**Comment by polar-kev** [comment]
@hferrone, can you please revert? We aren't done reviewing.

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md [hunk 1]
```diff
@@ -1,93 +1,289 @@
 # [Unity 2019/2020 + Windows XR Plugin](#tab/winxr)
 
-Once **MixedRealityFeatureTool** is opened click on start to get started with Mixed Reality Feature Tool.
+Once **MixedRealityFeatureTool** is opened, To access preview releases click on **Settings** and enable **Show preview releases** under **Feature** tab, then click on **ok** to save the settings.
 
-![MixedRealityFeatureTool window for Windows XR plugin](../images/mr-learning-base/base-02-section4-step1-2.png)
+![MixedRealityFeatureTool for legacy WSA](../images/mr-learning-base/base-02-section4-step1-2-preview.png)
```
Comment (CDiaz-MS) [inline mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:5]: I think this is an empty image link

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md [hunk 2]
```diff
@@ -1,93 +1,289 @@
 # [Unity 2019/2020 + Windows XR Plugin](#tab/winxr)
```
Comment (CDiaz-MS) [inline mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:1]: There are many empty image links on this page.

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/mr-learning-base-01.md [hunk 3]
```diff
@@ -51,10 +52,10 @@ If you installed Unity without these options, you can add them through **'Add Mo
 ![Unity Windows Build Support option](../../../develop/images/Unity_Install_Option_UWP2.png)
 
 > [!CAUTION]
-> The recommended MRTK version for this tutorial series is MRTK 2.6.
+> The recommended MRTK version for this tutorial series is MRTK 2.7.
 
 > [!CAUTION]
-> The recommended Unity version for this tutorial series is Unity 2019 LTS (currently 2019.4.x) . This supersedes any Unity version requirements stated in the prerequisites linked above.
+> This tutorial series supports Unity 2019 LTS (currently 2019.4.x) if you are using Legacy WSA or Window XR Plugin and also Unity 2020 LTS(currently 2020.3.x) if you are using Window XR Plugin or Open XR. This supersedes any Unity version requirements stated in the prerequisites linked above.
```
Comment (CDiaz-MS) [inline mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/mr-learning-base-01.md:58]: nit: "Windows XR Plugin"

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/configuring-profile.md [hunk 4]
```diff
@@ -1,60 +1,14 @@
...
-
-In the Clone Profile window, enter a suitable **Profile Name**, for example, _GettingStarted_XRSDKSpatialAwarenessSystemProfile_, then click the **Clone** button to create an editable copy of the **DefaultXRSDKSpatialAwarenessSystemProfile**:
-
-![Unity MixedRealityToolkit clone Spatial Awareness System Profile popup window](../images/mr-learning-base/base-03-section1-step3-2xrsdk.png)
-
-The newly created Spatial Awareness System Profile is now automatically assigned to your Configuration Profile:
-
-![Unity MixedRealityToolkit component with newly created custom MixedRealitySpatialAwarenessSystemProfile applied](../images/mr-learning-base/base-03-section1-step3-3xrsdk.png)
-
 ### 4. Clone the default Spatial Awareness Mesh Observer Profile
```
[the remark this answers is not in the document]
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/configuring-profile.md:3]: @CDiaz-MS  this document is the tab for 4th point in "mr-learning-base-03", after the 3rd point "3. Clone the default Spatial Awareness System Profile".

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md [hunk 5]
```diff
@@ -1,93 +1,289 @@
...
+> * Set default Spatial Awareness layer: Creates a Unity Layer named Spatial Awareness and configures MRTK to use this layer for the spatial awareness mesh. To learn more about Unity Layers, you can refer to Unity's <a href="https://docs.unity3d.com/Manual/Layers.html" target="_blank">Customizing Your Workspace</a> documentation.
+
+> [!TIP]
+> Setting the Audio spatializer property is optional but may improve the audio experience in your project. If you set it to MS HRTF Spatializer, this spatializer plugin will be used when Unity's AudioSource.spatialize property is enabled. To learn more about this topic, you can refer to the  <a href="//windows/mixed-reality/develop/unity/tutorials/unity-spatial-audio-ch1" target="_blank"> Spatial audio tutorials</a>.
+
+Click on **Next** then click on **Done** in the MRTK Project Configurator window to finish the Unity project configuration for XRSDK.
+
+### Configure additional project settings
+
+In the Unity menu, select **Edi**t > **Project Settings...** to open the Project Settings window:
```
[the remark this answers is not in the document]
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:79]: corrected

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md [hunk 6]
```diff
@@ -1,93 +1,289 @@
 # [Unity 2019/2020 + Windows XR Plugin](#tab/winxr)
 
-Once **MixedRealityFeatureTool** is opened click on start to get started with Mixed Reality Feature Tool.
+Once **MixedRealityFeatureTool** is opened, To access preview releases click on **Settings** and enable **Show preview releases** under **Feature** tab, then click on **ok** to save the settings.
```
[the remark this answers is not in the document]
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:3]: corrected..

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md (hunk 1 above)
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:5]: Image is present for this link,
image

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md [hunk 7]
```diff
@@ -1,93 +1,289 @@
...
+
+![MixedRealityFeatureTool for legacy WSA](../images/mr-learning-base/base-02-section4-step1-2-preview.png)
+
+next click on **Start** to get started with Mixed Reality Feature Tool.
 
-Under the **Platform Support** dropdown, select **Mixed Reality OpenXR Plugin**. Make sure the latest version is selected in the dropdown.
-![Selecting Mixed Reality OpenXR Plugin](../images/mrft-openxr.png)
+![MixedRealityFeatureTool for legacy WSA](../images/mr-learning-base/base-02-section4-step1-2.png)
+
+The fist step is to point the Mixed Reality Feature Tool to your **Project path** using the **ellipsis** button, click on the Three dots ellipsis button next to the Project path and browse to your project folder in the explorer for example _D:\MixedRealityLearning\MRTK Tutorials_.
```
[the remark this answers is not in the document]
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:108]: corrected

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md (hunk 1 above)
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:5]: corrected

File: mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md (hunk 2 above)
Comment (veeruby-developer) [reply mixed-reality-docs/mr-dev-docs/develop/unity/tutorials/includes/importing-mrtk.md:1]: corrected

**Pull request merged**